### ISSUE TITLE
crucible mounts the wrong data volume.

### ISSUE THREAD

**Issue by sofusalbertsen** [Praqma/staci#75]
Crucible container have stored its files under /opt/atlassian instead of the defaul /var/atlassian.
Therefore, Crucible image becomes statefull != good.

In order to change is, we need to edit bin/generateCompose.sh under crucible.

**Comment by hoeghh** [comment]
I dont understand.

In my local bin/generateCompose.sh it says : 

```
    - '$volume_dir'/crucible:/var/atlassian/crucible'
```

in line 102.

The one on github has the same line.

**Comment by sofusalbertsen** [comment]
Yes, but when you go into the container, the data is stored under /opt/atlassian/crucible, not /var

**Comment by hoeghh** [comment]
cr@p.... i will se when i can look into it.

**Comment by sofusalbertsen** [comment]
dont hurry, we have it under control here, but it is a must have feature :100:
